### PR TITLE
Add force default option to cast command

### PR DESCRIPTION
**Pull request by theychx**
as discussed

### CODE REVIEW

File: catt/cli.py [hunk 1]
```diff
@@ -89,9 +89,12 @@ def write_config(settings):
 
 @cli.command(short_help="Send a video to a Chromecast for playing.")
 @click.argument("video_url", callback=process_url)
+@click.option("-f", "--force-default", is_flag=True,
+              help="Force use of the default controller.")
```
Comment (skorokithakis) [inline catt/cli.py]: Can you explain a bit more what this does? End users probably won't know what a controller is.

Comment (theychx) [reply catt/cli.py]: Fair point, I considered using "default media controller" instead, but figured that that wouldn't be any clearer. Does such elaboration belong in cli help texts?. I can't think of a way to explain this in a few words. Any ideas?

Comment (skorokithakis) [reply catt/cli.py]: Hmm, yeah, good point. I can't find a way to summarize it either. Maybe if it's in terms of "what will happen"? What's a reason that would make someone want to use the default controller?

Comment (theychx) [reply catt/cli.py]: "default chromecast app", maybe?

Comment (skorokithakis) [reply catt/cli.py]: Yeah, that sounds better. Still, *why* would someone use that option? What's your use case for adding it? We can say something like "(use this if X doesn't work)".

Comment (theychx) [reply catt/cli.py]: > What's a reason that would make someone want to use the default controller?

If a custom controller is broken.

Comment (theychx) [reply catt/cli.py]: fixed

**Pull request merged**